### PR TITLE
add pod support

### PR DESCRIPTION
**Pull request by justinjpacheco**
- update the vagrant host definitions to include pod information
  - parse pod id out of hostname and use it to locate the correct pod
    networking information in the networking topology

### CODE REVIEW

File: chef/cookbooks/bcpc/attributes/networking.rb [hunk 1]
```diff
@@ -6,50 +6,73 @@
...
     }
-  }
+  ]
 }
 
 # try to get node information via the hostname
-# eg. ob-cloud-r(01)(a)n(01) = rack_id,pod_id,node_id
+# eg. cloud-r(01)(a)n(01) = rack_id,pod_id,node_id
 #
+rack_id,pod_id,node_id = [nil,nil,nil]
```
Comment (comay) [inline chef/cookbooks/bcpc/attributes/networking.rb]: Please add spaces after the commas on both sides of the assignment. It looks like there are other instances of this in the file too which should be corrected.

File: chef/cookbooks/bcpc/attributes/networking.rb [hunk 2]
```diff
@@ -6,50 +6,73 @@
...
+racks = default['bcpc']['networking']['topology']['racks']
 
 if match = node['hostname'].match(/.*r(\d+)(\w+)?n(\d+)$/i)
+
   rack_id,pod_id,node_id = match.captures
```
Comment (comay) [inline chef/cookbooks/bcpc/attributes/networking.rb]: For example, here. 😉

File: virtual/topology/1h1w.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
       }
     },
     {
-      "host": "r1n1",
+      "host": "r1an1",
```
Comment (comay) [inline virtual/topology/1h1w.json]: Oh noes, need to fight against muscle memory. 🙃

Comment (justinjpacheco) [reply virtual/topology/1h1w.json]: lol, i had the same thought

**Comment by justinjpacheco** [comment]
adding another example topology to show an additional pod wouldn't be a problem. just need to make sure the TOR can support it

**Pull request merged**